topics/single-cell/tutorials/scatac-standard-processing-snapatac2/tutorial.md

```diff
@@ -0,0 +1,940 @@
+>    - *"Method used for plotting"*: `Plot fragment size distribution, using 'pl.frag_size_distr'`
+>        - {% icon param-file %} *"Annotated data matrix"*: `Anndata 5k PBMC` (output of **pp.import_data** {% icon tool %})
+> 2. {% icon galaxy-eye %} Inspect the `.png` output
+> 3. {% tool [SnapATAC2 Plotting](toolshed.g2.bx.psu.edu/repos/iuc/snapatac2_plotting/snapatac2_plotting/2.5.3+galaxy1) %} with the following parameters:
+>    - *"Method used for plotting"*: `Plot fragment size distribution, using 'pl.frag_size_distr'`
+>        - {% icon param-file %} *"Annotated data matrix"*: `Anndata 5k PBMC` (output of **pp.import_data** {% icon tool %})
+>        - *"Change the y-axis (fragment counts) to log scale"*: `Yes`
+> 4. {% icon galaxy-eye %} Inspect the `.png` output
+>
+>
+>  ![fragment_size_distribution]({% link topics/single-cell/images/scatac-standard-snapatac2/pl.frag_size.png %})
+>
+>  ![log_fragment_size_distribution]({% link topics/single-cell/images/scatac-standard-snapatac2/log_pl.frag_size.png %})
+> > <question-title></question-title>
+> >
+> > 1. What distinct features do the plots have? And what do they represent?
+> > 2. Which fragments are generally from open chromatin?
+> >
+> > > <solution-title></solution-title>
+> > >
+> > > 1. 3 peaks are clearly visible (at <100-bp, ~200-bp and ~400-bp). The smallest fragments are from nucleosome-free regions, while the larger peaks (200- and 400-bp) contain mono- and di-nucleosome fragments, respectively. 
+> > > 2. The small fragments (<100-bp) are from open chromatin reads, since the Tn5 transposase could easily access the loosely packed DNA ({% cite Yan2020 %}). 
+> > > 
+> > {: .solution}
+> >
+> {: .question}
+{: .hands_on}
+
+
+The {TSSe} is another important {QC} metric. Nucleosome-free fragments are expected to be enriched at {TSS}. TSSe shows increased fragmentation of chromatin around the TSS. This suggests open and accessible nucleosome-free chromatin. 
+
+{TSSe} is used as a QC metric, since an increased enrichment around TSS regions suggests that the experiment has captured biological meaningful genomic features. 
+TSSe scores of individual cells can be calculated using SnapATAC2's *metrics.tsse* function. 
+
+> <hands-on-title> Calculate and Plot TSSe </hands-on-title>
+>
+> 1. {% tool [SnapATAC2 Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/snapatac2_preprocessing/snapatac2_preprocessing/2.5.3+galaxy1) %} with the following parameters:
+>    - *"Method used for preprocessing"*: `Compute the TSS enrichment score (TSSe) for each cell, using 'metrics.tsse'`
+>        - {% icon param-file %} *"Annotated data matrix"*: `Anndata 5k PBMC` (output of **pp.import_data** {% icon tool %})
+>        - {% icon param-file %} *"GTF/GFF file containing the gene annotation"*: `gene_annotation.gtf.gz` (Input dataset)
+>
+> 2. Rename the generated file to `Anndata 5k PBMC TSSe` or add the tag {% icon galaxy-tags %} `TSSe` to the dataset:
```

Comment:
No, but when I was using tutorials, I always added tags (the ones without '#') to the datasets instead of renaming them, since then you don't have to wait until the step is complete to continue with the next step. If you want, I can also remove the mentions of tags

Reply:
I just wanted to know. It doesn't hurt to have them.